### PR TITLE
Make text clearer by not changing colour over text

### PR DESCRIPTION
**Pull request by andrewrlee**
https://github.com/ministryofjustice/hmpps-developer-portal/assets/1517745/7241af97-03dd-41e5-adeb-c8f36dd05463

### CODE REVIEW

File: assets/scss/local.scss [hunk 1]
```diff
@@ -53,18 +53,30 @@
...
   display: flex;
   flex-wrap: wrap;
-  justify-content: center;
-  padding: 10px;
+  justify-content: left;
 
   .radiator-tile {
     flex: 0 0 320px;
-    margin: 1em;
+    margin: 10px 10px 10px 10px;
```
Comment (psoleckimoj) [inline assets/scss/local.scss:66]: can just do `margin: 10px;`

File: server/routes/driftRadiator.ts [hunk 2]
```diff
@@ -21,29 +42,23 @@ export default function routes({ serviceCatalogueService, redisService }: Servic
...
-    const queueInformation = req.params[0]
-
-    logger.info(`Queue call for ${componentName} with ${queueInformation}`)
-
-    const component = await serviceCatalogueService.getComponent(componentName)
-
-    const streams = component.environments
-      .map(e => `version:${component.name}:${e.name}`)
-      .map((key, id) => ({ key, id: `${id}` }))
+  post('/queue', async (req, res) => {
```
Comment (psoleckimoj) [inline server/routes/driftRadiator.ts:55]: Maybe we need to change all my `/queue` routes to this sort of thing?

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: assets/scss/local.scss (hunk 1 above)
Comment (andrewrlee) [reply assets/scss/local.scss:66]: Will fix in a follow up

**Pull request merged**